Android.mk

```diff
@@ -200,7 +201,7 @@ $(TARGET_BLOBS_SHA1_OUT):
 .PHONY: $(TARGET_BLOBS_MAP)
 $(TARGET_BLOBS_MAP): $(TARGET_BLOBS_SHA1_OUT)
 	@rm -f $@ && touch $@; \
-	grep ':' $(TARGET_DEVICE_BLOBS) | grep -v '^\s*#' | cut -d' ' -f1 | grep ':' | while read line; \
+	grep -h ':' $(TARGET_DEVICE_BLOBS) | grep -v '^\s*#' | sed -e 's/^ *//' | sed -e 's/ *\\$$//' | grep ':[^=]' | while read line; \
```

Comment:
Might be worth a comment to explain the new behavior. Especially since we might rely on GNU tools specifics arguments.